playbooks/aws/README.md

```diff
@@ -188,12 +157,46 @@ openshift_hosted_registry_wait=False
 
 There are more examples of cluster inventory settings [`here`](../../inventory/byo/).
 
+In order to create the bootstrapable AMI we need to create an openshift-ansible inventory file.  This file enables us to create the AMI using the openshift-ansible node roles.
+
+
+#### Step 2
+
+Once the vars.yml file has been updated with the correct settings for the desired AWS account then we are ready to build an AMI.
```

Comment:
What vars.yml file?  If you're referring the users to make in-tree changes, I don't think this is a good approach.